stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/startup/StartupFragment.kt

```diff
@@ -0,0 +1,22 @@
+package io.getstream.chat.sample.feature.startup
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.sample.R
+import io.getstream.chat.sample.common.navigateSafely
+
+class StartupFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        findNavController().navigateSafely(
+            if (ChatClient.isInitialized && ChatClient.instance().getCurrentUser() != null) {
```

Comment:
How come we didn't add this logic to `ChatInitializer` in this sample app, but we did in the other?

Reply:
🙈 nice catch, forgot to refactor that